logstash/README.md

```diff
@@ -0,0 +1 @@
+# ![](https://github.com/signalfx/integrations/blob/master/logstash/img/integration_logstash.png) Logstash
```

Comment:
this link is 404

Reply:
this is because the image points to master. link will be fixed once changes are in master